ui/js/graphs.js

```diff
@@ -801,6 +814,71 @@ perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
         });
     };
 
+    var resolvePromise = $q.defer();
+    var addRelatedPlatforms = function(relatedSeries) {
+        PhSeries.getAllSeries(relatedSeries.projectName,
+            $scope.timeRange, optionCollectionMap).then(
+            function(seriesData) {
+                var platformList = seriesData.platformList;
```

Comment:
I still use the same method to get platform list rather than the endpoint in bug 1189519, because not only platform but also some other data(e.g: seriesList and signature) I need get from remote.